packages/web-components/fast-components/src/slider/fixtures/base.html

```diff
@@ -33,6 +33,50 @@
         <!-- Margin offsets -->
         <fast-slider style="margin-left:100px"></fast-slider>
 
+        <!-- Negative positions -->
```

Comment:
It could be done in a separate PR, but I believe in many of the test templates we have headings. Perhaps that would help visually break this up and provide more context than just the comments?

Reply:
ok will do, I also think now there are some labels not using the design system so in fast-components they are dark text on default dark mode. Will fix both in a separate PR